packages/account/src/Components/poi-poa-docs-submitted/poi-poa-docs-submitted.jsx

```diff
@@ -1,21 +1,39 @@
 import React from 'react';
-import { Button, Icon } from '@deriv/components';
+import { Button, Icon, Loading } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { getAuthenticationStatusInfo } from '@deriv/shared';
 
 import IconMessageContent from 'Components/icon-message-content';
 
-const PoiPoaDocsSubmitted = ({ account_status, is_vanuatu_selected, onClickOK, updateAccountStatus }) => {
+const PoiPoaDocsSubmitted = ({
+    account_status,
+    jurisdiction_selected_shortcode,
+    onClickOK,
+    updateAccountStatus,
+    has_created_account_for_selected_jurisdiction,
+    openPasswordModal,
+}) => {
+    const [is_loading, setIsLoading] = React.useState(false);
     React.useEffect(() => {
-        updateAccountStatus();
+        setIsLoading(true);
+        updateAccountStatus().then(() => {
+            setIsLoading(false);
+        });
+
         //eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    const message = localize('Your documents were submitted successfully');
+    const onSubmit = () => {
+        onClickOK();
+        if (!has_created_account_for_selected_jurisdiction) {
+            openPasswordModal();
+        }
+    };
+
     const getDescription = () => {
         const { manual_status, poi_verified_for_vanuatu, poi_verified_for_bvi_labuan_maltainvest, poa_pending } =
             getAuthenticationStatusInfo(account_status);
-
+        const is_vanuatu_selected = jurisdiction_selected_shortcode === 'vanuatu';
```

Comment:
Just a reminder, we need to create a task to move hardcoded code to a constants file.

Reply:
sure